content/docs/wrp/simple-messages.md

```diff
@@ -47,9 +54,10 @@ the same message type.
 Name | Description
 -----|--------------
 msg_type | The message type for the request-response.  (**SHALL** be 3.)
-source | The device_id name of the device originating the request or response.
-dest | The device_id name of the target device of the request or response.
-content_type | (optional) The media type of the payload.
+source | The device_id name of the device originating the request or response (i.e. dns:tr1d1um.example.net for inbound messages).
+dest | The device_id name of the target device of the request or response (i.e. mac:112233445566/config for inbound messages).
```

Comment:
```suggestion
dest | The locator information of the target device of the request or response (i.e. mac:112233445566/config for inbound messages).
```
I think calling this `device_id` is confusing.
Locator is a reference to this page: https://xmidt.io/docs/wrp/basics/#locators